modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/LoaderMessage.scala

```diff
@@ -117,10 +117,11 @@ object LoaderMessage {
             grouped.sorted.lastOption
           }
 
-      implicit val snowplowEntityOrdering: math.Ordering[SnowplowEntity] = math.Ordering.by(_.toString)
 
       implicit val wideRowTypeOrdering: math.Ordering[WideRow.Type] = {
+        implicit val snowplowEntityOrdering: math.Ordering[SnowplowEntity] = math.Ordering.by(_.toString)
```

Comment:
I might be prematurely optimizing, but it bothers me a bit that we keep on serializing the same thing to string again and again.  How about

```
math.ordering.by {
  case SnowplowEntity.Context => 0
  case SnowplowEntity.SelfDescribingEvent => 1
}
```